tse_utils/duplicates.go

```diff
@@ -26,13 +27,16 @@ func RemoveDuplicates(candidates []*RegistroTSE, fileBeingHandled string) (map[s
 		foundCandidate := candidatesMap[c.CPF]
 		if foundCandidate == nil { // candidate not present on map, add it
 			fileLines++
-			nascimentoAsTime, err := time.Parse("02/01/2006", c.CandidatoTSE.Nascimento)
-			if err != nil {
-				return nil, fmt.Errorf("falha ao fazer parse de data de nascimento de candidato %s para time.Time, erro %q", c.CandidatoTSE.Nascimento, err)
+			var nascimentoAsTimestamp *timestamppb.Timestamp
```

Comment:
Qual é o timestamp aqui? 0? Pode conferir por favor? Se o valor zero para um timestamp for time.Now, não atingiremos nosso objetivo.

Reply:
é nil, verifiquei aqui